File: Fibonacci.js
// Print the first 15 digits of the fibonacci sequence

let fib = (n) => {
    let fibList = [0,1];
    for (let i=2; i<n; i++) {
        fibList.push(fibList[i-1] + fibList[i-2])
    }
    return fibList
}

console.log(fib(15));

File: Fibonacci.py
#Print the first 15 digits of the fibonacci sequence

def find_fib(n):
    fib = [0,1]
    for i in range(2, n):
        fib.append(fib[i-1] + fib[i-2])
    return fib

print(find_fib(15))